### PR TITLE
Added an index

### PR DESCRIPTION
**Pull request by navanchauhan**
Fixed the conflicts of #471

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,19 @@ More details about markers:
 * [Artoolkit Open Doc](https://github.com/artoolkit/artoolkit-docs/tree/master/3_Marker_Training)
 * [Detailed Article about markers](https://medium.com/chialab-open-source/ar-js-the-simpliest-way-to-get-cross-browser-ar-on-the-web-8f670dd45462) by [@nicolocarpignoli](https://github.com/nicolocarpignoli)
 
+# Index
+* [Get Started](#Get-Started)
+* [Guides for Beginners](#Guides-for-beginners)
+* [Advance Guides](#Advance-Guides)
```
Comment (janpio) [inline README.md]:
```suggestion
* [Advanced Guides](#Advanced-Guides)
```

**Comment by navanchauhan** [comment]
@janpio I think I added all the major headings, if I missed anything you can update it

File: README.md (hunk 1 above)
Comment (nicolocarpignoli) [reply README.md]: Once solved this , for me it is ok

Comment (nicolocarpignoli) [reply README.md]: @navanchauhan please solve this :)

**Comment by navanchauhan** [comment]
@nicolocarpignoli

**Pull request merged**